DTAConfig/Settings/FileSourceDestinationInfo.cs

```diff
@@ -151,6 +134,27 @@ public void Revert()
                         $"Invalid {nameof(FileOperationOptions)} value of {FileOperationOptions}");
             }
         }
+
+#if NETFRAMEWORK
+        [DllImport("Kernel32.dll", CharSet = CharSet.Unicode)]
+        private static extern bool CreateHardLink(
+            string lpFileName,
+            string lpExistingFileName,
+            IntPtr lpSecurityAttributes
+        );
+#endif
+
+        /// <summary>
+        ///  Creates a symbolic link from the source file to the destination file.
+        /// </summary>
+        private void CreateSymbolicLink(string source, string destination)
+        {
+#if NETFRAMEWORK
+            CreateHardLink(destination, source, IntPtr.Zero);
```

Comment:
Apart from the comments above, symlinks and hard links are two different concepts. You should call `CreateSymbolicLinkW` to make this function correctly behaves

Reply:
> thing is, you can't create symlinks on Windows as a regular user, we're currently investigating if that is solvable by adequate means

At least one should never create a hard link in a function explicitly named `CreateSymbolicLink`